src/debugAdapter/Readme.md

```diff
@@ -13,6 +13,6 @@ cd vscode-go
 npm install
 ```
 2. Open the `vscode-go` folder in one instance.  Choose the `Launch Extension` debug target and hit F5 to launch a second instance.
-3. In the second instance, open the Go application you'd like to test against.  In that instance, create a new Go debug target pointing at the program you want to debug, and add `"debugServer": 4711` in the root of the configuration.
+3. In the second instance, open the Go application you'd like to test against.  In that instance, create a new Go debug target pointing at the program you want to debug, and add `"debugServer": 4712` in the root of the configuration.
```

Comment:
Why 4712?

Reply:
Because of this https://github.com/Microsoft/vscode-go/blob/1cdd3667e957c1aa080f0b6f987771d436b9db6a/.vscode/launch.json#L21 it starts the server at 4712 not 4711

Reply:
4711 is used in https://github.com/Microsoft/vscode-go/blob/dc7c4739b7923c206ac9cc8b5ef95fa874cb17bd/src/debugAdapter/.vscode/launch.json 

Let's use 4711 at https://github.com/Microsoft/vscode-go/blob/1cdd3667e957c1aa080f0b6f987771d436b9db6a/.vscode/launch.json#L21 as well